ui/src/components/Views/Configs/MountPointForm.tsx

```diff
@@ -116,11 +116,12 @@ export const OnBoarding = (): ReactElement => {
             Select where LocalStack will be mounted
           </Typography>
           <Typography variant='subtitle2'>
-            {`For MacOS users it will be under /Users/${userState.selectedUser || 'loading...'}/.localstack-volume`}
+            {`For MacOS users it will be under /Users/${userState.selectedUser || 'loading...'} \
+            /.cache/localstack/volume`}
           </Typography>
           <Typography variant='subtitle2' gutterBottom>
             {`For Linux/Windows users it will be under \
-             /home/${userState.selectedUser || 'loading...'}/.localstack-volume`}
+             /home/${userState.selectedUser || 'loading...'}/.cache/localstack/volume`}
```

Comment:
Nice improvement - great that you found out that Docker can mount non-existing folders! 🎉 (such that we can mount from the host `~/.cache` here)